Autocoders/MagicDrawCompPlugin/src/gov/nasa/jpl/componentaction/MainMenuConfigurator.java

```diff
@@ -70,7 +70,7 @@
  * <br>
  * \ref IsfComponentAction.java - NOT USED.
  * <br>
- * \ref LoadIDCOnfigAction.java - Creates button to load id configuration file within the "Component Autocoder" tab.
+ * \ref LoadIDConfigAction.java - Creates button to load id configuration file within the "Component Autocoder" tab.
```

Comment:
I usually try to make sure that my changes match the underlying file system, but this PR was quite large and I neglected to do that. In this case, the repository filename is correctly cased -- i.e. it's just the comment that suffered from a stuck `<shift>` key.